docs/rationale.md

```diff
@@ -13,7 +13,7 @@ The first requirement of Prettier is to output valid code that has the exact sam
 
 ### Strings
 
-Double or single quotes? Prettier chooses the one which results in the fewest number of escapes. `"It's gettin' better!"`, not `'It\'s gettin\' better!'`. In case of a tie, Prettier defaults to double quotes (but that can be changed via the [`--single-quote`](options.html#quotes) option).
+Double or single quotes? Prettier follows the JavaScript ecosystem, which has largely standardized on single quotes, with it being the most popular style seen in the wild.
```

Comment:
The 'fewest number of escapes' part shouldn't get removed. Please keep it somehow.

Reply:
Ah, sorry! I misunderstood that this was describing a separate feature. I've added this back.